nncf/experimental/torch/fx/nncf_graph_builder.py

```diff
@@ -115,16 +131,13 @@ def create_nncf_graph(model: torch.fx.GraphModule) -> PTNNCFGraph:
         :param model: torch fx GraphModule.
         :return: NNCFGraph.
         """
-
         nncf_graph = PTNNCFGraph()
-
         for source_node in model.graph.nodes:
             node_type, node_metatype = GraphConverter._get_node_type_and_metatype(source_node, model)
-
+            node_metatype = GraphConverter._map_fx_unique_metatypes(source_node, node_metatype)
+            is_shared_node = len(source_node.users) > 1 if source_node.op in ("get_attr",) else False
```

Comment:
Please do not count on a transformation here, check all possible cases

Reply:
Done!